### ISSUE TITLE
WinUI 3  HyperlinkButton NavigateUri not working

### ISSUE THREAD

**Issue by ghost1372** [microsoft/microsoft-ui-xaml#5630]
**Describe the bug**
I want to open Windows settings But the following code does not work And nothing happens

`<HyperlinkButton Content="Windows color settings" NavigateUri="ms-settings:personalization-colors" />
`
**Steps to reproduce the bug**

**Expected behavior**
Open the given address

**Screenshots**

**Version Info**

NuGet package version: 
```
<PackageReference Include="Microsoft.ProjectReunion" Version="0.8.1" />
    <PackageReference Include="Microsoft.ProjectReunion.Foundation" Version="0.8.1" />
    <PackageReference Include="Microsoft.ProjectReunion.WinUI" Version="0.8.1" />
```

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
| No (Worked) | Yes (Not Worked) |

| Windows version                    | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| October 2020 Update (19042)        | Yes   |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

**Comment by StephenLPeters** [comment]
@ghost1372 does this type of navigation work in your app, just not when used with a hyperlink button? Or is opening the settings app at all not working?

**Comment by StephenLPeters** [comment]
@Austin-Lamb and @MikeHillberg FYI,  Transfer to Windows App SDK?

**Comment by codendone** [comment]
It is possible the fix for #4438 might also fix this case.

**Comment by ghost1372** [comment]
@StephenLPeters  I tested with **HyperLink**
```
<TextBlock>
          <Run Text="Go to " />
              <Hyperlink NavigateUri="ms-settings:personalization-colors">
              </Hyperlink>
        </TextBlock>
```
and settings Could not be opened again.
Of course, the following code works well and opens the settings
`bool result = await Windows.System.Launcher.LaunchUriAsync(new Uri("ms-settings:personalization-colors"));
`

**Comment by Austin-Lamb** [comment]
It seems likely this will also be fixed by #4438 so closing this as a duplicate of that one.
